integration/spark-common-test/src/test/scala/org/apache/carbondata/spark/testsuite/datacompaction/TableLevelCompactionParameterTest.scala

```diff
@@ -0,0 +1,183 @@
+/*
```

Comment:
This test case is same as TableLevelCompactionOptionTest.scala, please remove this one(TableLevelCompactionParameterTest.scala)